src/drafts/hooks/useDynamicTextareaHeight.ts

```diff
@@ -16,43 +16,47 @@ type UseDynamicTextareaHeightSettings = {
  * resizing it as the user types. If the user manually resizes the textarea, their setting
  * will be respected.
  *
- * Returns an object to spread to the component's `sx` prop.
+ * Returns an object to spread to the component's `sx` prop. If you are using `Textarea`,
+ * apply this to the child `textarea` element: `<Textarea sx={{'& textarea': resultOfThisHook}} />`.
  *
  * NOTE: for the most accurate results, be sure that the `lineHeight` of the element is
  * explicitly set in CSS.
  */
 export const useDynamicTextareaHeight = ({
   minHeightLines,
   maxHeightLines,
-  element,
+  elementRef,
   value,
 }: UseDynamicTextareaHeightSettings): SxProp['sx'] => {
   const [height, setHeight] = useState<string | undefined>(undefined)
   const [minHeight, setMinHeight] = useState<string | undefined>(undefined)
   const [maxHeight, setMaxHeight] = useState<string | undefined>(undefined)
 
   useLayoutEffect(() => {
+    const element = elementRef.current
```

Comment:
Would it worth to check if `elementRef` is not null here or not really?

Reply:
It shouldn't ever be `null` since it's a required prop - I think ESLint will actually complain about an unnecessary check in that case